C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/Point2DCollection.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace PE11VectorDistanceCalculation
 {
+    //Not sure the point of this class.
+    //I feel like you should be able to accomplish everything in this class in a method in main.cs
```

Comment:
I typically try to keep the main method clean... not sure if that is something desirable or not or where I picked it up.

Reply:
Fair, but its a small application. No reason to make things overly complicated. The fact that you have to implement this type of class to handle essentially two identical objects makes me weary and should be an indicator to you that refactoring is necessary. I dont hate the class but just food for thought.

Reply:
I agree.  I know I was violating DRY as I was creating a second class that was virtually identical (which was why I was trying to turn it into one generic class)... :(